domain/domain.go

```diff
@@ -93,24 +94,25 @@ func (do *Domain) loadInfoSchema(handle *infoschema.Handle, usedSchemaVersion in
 	// Update self schema version to etcd.
 	defer func() {
 		if err != nil {
-			log.Info("[ddl] not update self schema version to etcd")
+			log.Info("cannot update self schema version to etcd.")
 			return
 		}
 		err = do.ddl.SchemaSyncer().UpdateSelfVersion(context.Background(), latestSchemaVersion)
 		if err != nil {
-			log.Infof("[ddl] update self version from %v to %v failed %v", usedSchemaVersion, latestSchemaVersion, err)
+			log.Info("update self version failed.", zap.Int64("usedSchemaVersion", usedSchemaVersion),
+				zap.Int64("latestSchemaVersion", latestSchemaVersion), zap.Error(err))
 		}
 	}()
 
 	startTime := time.Now()
 	ok, tblIDs, err := do.tryLoadSchemaDiffs(m, usedSchemaVersion, latestSchemaVersion)
 	if err != nil {
 		// We can fall back to full load, don't need to return the error.
-		log.Errorf("[ddl] failed to load schema diff err %v", err)
+		log.Error("failed to load schema diff", zap.Error(err))
 	}
 	if ok {
-		log.Infof("[ddl] diff load InfoSchema from version %d to %d in %v, tableIDs %v",
-			usedSchemaVersion, latestSchemaVersion, time.Since(startTime), tblIDs)
+		log.Info("diff load InfoSchema from version failed.", zap.Int64("usedSchemaVersion", usedSchemaVersion),
+			zap.Int64("latestSchemaVersion", latestSchemaVersion), zap.Any("time", time.Since(startTime)), zap.Any("tblIDs", tblIDs))
```

Comment:
It seems that we can use `Duration` instead of the first`Any`, and `Uint64s` instead of the second `Any`.

Reply:
Dony.
I use `Duration` and `Int64s`